legal-api/src/legal_api/reports/report.py

```diff
@@ -1387,6 +1387,9 @@ class ReportMeta:  # pylint: disable=too-few-public-methods
         'affidavit': {
             'documentType': 'affidavit'
         },
+        'directorAffidavit': {
+            'documentType': 'director_affidavit'
+        },
```

Comment:
I'd like to see plural directors, ie. `directorsAffidavit` and `directors_affidavit`.

This is line with what the UI labels this field.

Reply:
I referred UX design to name this. It says `Director Affidavit`
![image](https://github.com/bcgov/lear/assets/64869009/5f576088-ae60-4477-9ae7-876d40782584)

Reply:
Sure, no problem. The UI shows something different, hahaha.